### PR TITLE
add getWorstMoves

### PR DESCRIPTION
**Pull request by benjaminpjones**
This function is a little smarter than compute worst moves as it determines whether to use score on its own, and also filters out non-blunders.

### CODE REVIEW

File: src/AIReview.ts [hunk 1]
```diff
@@ -76,3 +76,26 @@ export function computeWorstMoves(starting_move:MoveTree,
...
+        threshhold = -0.2;
+    }
+
+    const filtered_worst_moves = worst_moves.filter(de => de.delta <= threshhold);
+
+    if (filtered_worst_moves.length >= 3) {
+        return filtered_worst_moves;
+    }
+
+    return worst_moves.slice(0, 3);
```
Comment (anoek) [inline src/AIReview.ts:100]: Since we're returning a potentially long array `filtered_worst_moves`, seems like we shouldn't be slicing `worst_moves`?

**Review by anoek — COMMENTED** [review]
Lets remove the `export` from `computeWorstMoves`, it seems like `getWorstMoves` is going to be the function anyone wants to call. Alternatively, rename both to keep the interface consistent, although `getWorstMoves` is probably a better name anyways.

File: src/AIReview.ts (hunk 1 above)
Comment (benjaminpjones) [reply src/AIReview.ts]: I think this is by design?  So we only return that slice in the event that the filter was too strong (say only 2pt losses the entire game)

**Comment by benjaminpjones** [comment]
Went ahead and removed the export from `computeWorstMoves`.  Left the name as-is as I think `getWorstMoves` is different enough in what it returns.

**Comment by anoek** [comment]
Thanks for the improvement!

**Pull request merged**